pf/tfbridge/provider_update.go

```diff
@@ -96,10 +96,15 @@ func (p *provider) Update(
 	}
 
 	// TODO handle resp.Private
```

Comment:
should we file a ticket for this

Reply:
This is https://github.com/pulumi/pulumi-terraform-bridge/issues/747 but you are correct that it should be cross-linked to the TODO. I'll do a sweep to cross-link TODOs as a separate PR.